app/scripts/controllers/createClientBuild.js

```diff
@@ -131,6 +131,14 @@ angular.module('openshiftConsole')
         };
       }
 
+      if (clientConfig.envVars) {
+        _.forEach(clientConfig.envVars, function(envVar) {
+          if (envVar.name) {
```

Comment:
@sedroche we still need to check for empty key value pairs as the YAML file won't accept this.